x-pack/plugins/apm/ftr_e2e/cypress/integration/read_only_user/transactions_overview/transactions_overview.spec.ts

```diff
@@ -35,8 +36,18 @@ describe('Transactions Overview', () => {
     cy.loginAsReadOnlyUser();
   });
 
+  it('has no detectable a11y violations on load', () => {
+    cy.visit(serviceTransactionsHref);
+    cy.contains('a[role="tab"]', 'Transactions').should(
+      'have.class',
```

Comment:
We might want to avoid depending on eui classed. If we need to check that a tab is selected, we could instead use aria-selected="true" for example.